Library/Homebrew/test/utils/analytics_spec.rb

```diff
@@ -0,0 +1,123 @@
+require "utils/analytics"
+
+describe Utils::Analytics do
+  describe "::os_prefix_ci" do
+    context "when anonymous_os_prefix_ci is not set" do
+      it "returns OS_VERSION and prefix when HOMEBREW_PREFIX is not /usr/local" do
+        h_prefix = HOMEBREW_PREFIX
+        HOMEBREW_PREFIX = mktmpdir/"blah"
+
+        begin
+          expect(described_class.os_prefix_ci).to include("#{OS_VERSION}, non-/usr/local")
+        rescue
```

Comment:
I think this `rescue` is unnecessary, since everything's handled in the `ensure` block.